apps/web/src/design-system/cards/Cards.tsx

```diff
@@ -48,15 +52,20 @@ function Card({ cell }: { cell: ICardCell }) {
   };
 
   return (
-    <StyledCard dark={colorScheme === 'dark'} onClick={handleOnClick}>
+    <StyledCard
+      dark={colorScheme === 'dark'}
+      onClick={cell.onClick ?? handleOnClick}
```

Comment:
I think this logic should be moved to inside `handleOnClick`.
```suggestion
      onClick={handleOnClick}
```
And inside of the method:
```
const handleOnClick = () => {
    if (cell.onClick) {
       onClick();
       return;
     }
     
     if (cell.navigateTo) {
       navigate(cell.navigateTo);
     }
  };
```